static/app/components/performance/searchBar.tsx

```diff
@@ -43,6 +44,20 @@ function SearchBar(props: SearchBarProps) {
 
   const projectIdStrings = (eventView.project as Readonly<number>[])?.map(String);
 
+  useEffect(() => {
+    const handleBackgroundPointerUp = (e: PointerEvent) => {
+      if (containerRef.current?.contains(e.target as Node)) {
+        return;
+      }
+
+      closeDropdown();
+    };
+
+    document.addEventListener('pointerup', handleBackgroundPointerUp);
+
+    return () => document.removeEventListener('pointerup', handleBackgroundPointerUp);
+  }, [containerRef]);
+
```

Comment:
Shouldn't the presence of the dropdown just be connected to if the input has focus?

Reply:
@evanpurkhiser Doesn't seem to work that way, `SearchDropdown` doesn't close automatically if the input loses focus so we need to manually handle it. This is done the same way [here](https://github.com/getsentry/sentry/blob/0db4169dd7e27a273a00c903cf96f5048f24031b/static/app/components/smartSearchBar/index.tsx#L423)

Reply:
Ah ok I see, this is actually setting up it's own search bar. I thought this was actually using the SmartSearchabr